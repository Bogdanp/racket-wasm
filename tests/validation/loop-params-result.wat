(module
 (func
  (i32.const 1)
  (i64.const 2)
  (loop
   (param i32)
   (param i64)
   (result i32)
   (drop)
   (br 1))))
