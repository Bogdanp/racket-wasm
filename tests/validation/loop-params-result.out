(#f . "result arity error\n  expected: [i32]\n  found: []\n  in: loop-params-result.wasm\n  offset: 34 (0x22)\n  at: (instr:loop #<loc> (functype '(i64 i32) '(i32)) (vector (instr:drop #<loc>) (instr:br #<loc> 1)))\n  within: (funcidx 0)")